airflow/providers/microsoft/teams/operators/teams_webhook.py

```diff
@@ -0,0 +1,69 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+from typing import TYPE_CHECKING, Optional
+
+from airflow.providers.http.operators.http import SimpleHttpOperator
+from airflow.providers.microsoft.teams.hooks.teams_webhook import TeamsWebhookHook
+
+if TYPE_CHECKING:
+    from airflow.utils.context import Context
+
+
+class TeamsWebhookOperator(SimpleHttpOperator):
+    def __init__(
+        self,
+        http_conn_id: str,
+        webhook_token: Optional[str] = None,
+        message: str = "",
+        subtitle: Optional[str] = None,
+        theme_color: Optional[str] = None,
+        facts: Optional[list] = None,
+        action_button_name: Optional[str] = None,
+        action_button_url: Optional[str] = None,
+        icon_url: Optional[str] = None,
+        proxy: Optional[str] = None,
+        **kwargs,
+    ):
+        super().__init__(endpoint=webhook_token, **kwargs)
+        self.http_conn_id = http_conn_id
+        self.webhook_token = webhook_token
+        self.message = message
+        self.subtitle = subtitle
+        self.theme_color = theme_color
+        self.facts = facts
+        self.action_button_name = action_button_name
+        self.action_button_url = action_button_url
+        self.icon_url = icon_url
+        self.proxy = proxy
+        self.hook: Optional[TeamsWebhookHook] = None
+
+    def execute(self, context: 'Context') -> None:
+        self.hook = TeamsWebhookHook(
+            self.http_conn_id,
+            self.webhook_token,
+            self.message,
+            self.subtitle,
+            self.theme_color,
+            self.facts,
+            self.action_button_name,
+            self.action_button_url,
+            self.icon_url,
+            self.proxy,
+        )
+        self.hook.send()
```

Comment:
```suggestion

    def execute(self, context: 'Context') -> None:
        hook = TeamsWebhookHook(
            self.http_conn_id,
            self.webhook_token,
            self.message,
            self.subtitle,
            self.theme_color,
            self.facts,
            self.action_button_name,
            self.action_button_url,
            self.icon_url,
            self.proxy,
        )
        hook.send()
```
Since this object is only used in one method, IMO it doesn't make sense for it to be an instance attr.